### ISSUE TITLE
[Bug]: Mysql and Redis not working

### ISSUE THREAD

**Issue by crgAdrian** [beyondcode/herd-community#813]
### Platform

Windows

### Operating system version

Windows 10 PRO 22H2

### System architecture

Intel (x86)

### Herd Version

1.7.1

### PHP Version

PHP 8.2

### Bug description

Hi, since last update i can't run mysql and redis services

```
2024-06-17T08:01:45.055441Z 1 [ERROR] [MY-012271] [InnoDB] The innodb_system data file 'ibdata1' must be writable
2024-06-17T08:01:45.055870Z 1 [ERROR] [MY-012278] [InnoDB] The innodb_system data file 'ibdata1' must be writable
2024-06-17T08:01:45.054223Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2024-06-17T08:01:45.055441Z 1 [ERROR] [MY-012271] [InnoDB] The innodb_system data file 'ibdata1' must be writable
2024-06-17T08:01:45.055870Z 1 [ERROR] [MY-012278] [InnoDB] The innodb_system data file 'ibdata1' must be writable
2024-06-17T08:01:45.056331Z 1 [ERROR] [MY-010334] [Server] Failed to initialize DD Storage Engine
2024-06-17T08:01:45.057113Z 0 [ERROR] [MY-010020] [Server] Data Dictionary initialization failed.
```

```
[018536] 17 Jun 10:59:41.948 * RedisWin is starting ......
[018536] 17 Jun 10:59:41.948 * RedisWin version=7.2.4, bits=64, commit=14/01/2024, modified=0, pid=18536, just started
[018536] 17 Jun 10:59:41.948 * Configuration loaded
[018536] 17 Jun 10:59:41.949 * monotonic clock: X64 clock_gettime
[018536] 17 Jun 10:59:41.950 * Running mode=standalone, port=6379.
[018536] 17 Jun 10:59:41.951 # Warning: Could not create server TCP listening socket *:6379: bind: No such file or directory
[018536] 17 Jun 10:59:41.951 # Failed listening on port 6379 (tcp), aborting.
```

**Comment by sschlein** [comment]
Hi @crgAdrian could you check if there are stale services still running? Stopping and restarting all from the dashboard should do the trick.

There's currently a problem with that and we're working on a fix.

**Comment by crgAdrian** [comment]
@sschlein i created a new mysql service with another port and its working.

Another weird situation:
- i'm using workbech for mysql and i can connect on both ports 3307, 3306 even if 3306 is down.

**Comment by sschlein** [comment]
I am pretty sure the're a stale process on this port. Herd can't reuse the process and so it can't start a new one on the port and thinks it's not working.

**Comment by sschlein** [comment]
Fixed in 1.7.2
